velox/common/file/FileSystems.cpp

```diff
@@ -61,6 +61,16 @@ std::shared_ptr<FileSystem> getFileSystem(
   VELOX_FAIL("No registered file system matched with file path '{}'", filePath);
 }
 
+bool isPathSupportedByRegisteredFileSystems(const std::string_view& filePath) {
+  const auto& filesystems = registeredFileSystems();
+  for (const auto& p : filesystems) {
```

Comment:
nit: `const auto& [schemeMatcher, fileSystemGenerator] : filesystems`

Reply:
I see we don't use this for the existing `getFileSystem()` API as well. So this change is not required.